### ISSUE TITLE
It works, but it's not fast

### ISSUE THREAD

**Issue by loveqyx** [MisterHuhua/cloudflare_translate#1]
It is relatively slow to use in China, there is no speed, but it is very good and supports many languages. I am also ready to try to build one by myself

**Comment by HuayongHu** [comment]
Indeed, because you're using Cloudflare's network, it's not fast to use domestically.
